pwnlib/log.py

```diff
@@ -663,8 +663,6 @@ def install_default_handler():
     the ``pwnlib`` root logger.  This function is automatically called from when
     importing :mod:`pwn`.
     '''
-    console.stream = sys.stdout
```

Comment:
Looks like this was accidentally keeping our unit-tests from breaking by re-invoking `install_default_handler()` for each module's doctests.  The root-cause is `pwnlib.commandline.common` was setting it to `stderr`, which should only happen when `main()` is invoked, not when the module is imported.